src/utils/displayFunctions.ts

```diff
@@ -3,16 +3,15 @@ import {
   stringifyRatio,
   stringifyValue,
 } from '@agoric/ui-components';
-import { AssetKind } from '@agoric/ertp';
+import { AssetKind, AmountMath } from '@agoric/ertp';
+import {
+  floorMultiplyBy,
+  makeRatioFromAmounts,
+} from '@agoric/zoe/src/contractSupport';
```

Comment:
this should be declared in package.json as a dependency. I guess we don't have the lint rule for that yet. 

https://github.com/Agoric/dapp-inter/pull/42 takes care of adding the dep.

Reply:
Rebased